src/lib/components/ui/ArticleList.svelte

```
@@ -138,56 +140,56 @@
+				<ArticleCard
+					{article}
+					onBadgeClick={(category) => {
+						selectedCategory = category;
+						scrollToLatestResearch();
+					}}
+				/>
+			</div>
+		{/each}
+
+		{#if loading}
+			{#each Array(articlesPerPage) as _}
```

ESLint

[error] '_' is defined but never used.

(@typescript-eslint/no-unused-vars)